src/pii_extract_plg_regex/modules/fr/fr/phone_number.py

```diff
@@ -0,0 +1,66 @@
+"""
+FR phone numbers
+"""
+
+import re
+from typing import Iterable, Tuple
+
+import phonenumbers
+from pii_data.types import PiiEnum
+
+# ----------------------------------------------------------------------------
+
+# Regex for mobile & landline phone numbers
+PHONE_REGEX = r"""
+    \b
+    0\s*[1-9]             # 0 + Area code (1-9)
+    (?:[\s.-]*\d{2}){4}   # Remaining 8 numbers (all together or by pairs)
+    \b
+"""
+
+# Context that must be found around the phone number
+CONTEXT_REGEX = r"""
+    \b
+    t[ée]l[ée]phon((es?)|iques?) |                      # Phone & corresponding adjective
+    t[eé]l \. |                                         # Abbreviation
+    mobiles? |                                          # Mobile
+    appell?(er|âm|ât|èr)?a?i?e?o?(s|t|ns|z|nt|é|)? |    # Conjugation of 'to call'
+    coups? de fil |                                     # Informal synonym of call
```

Comment:
I think this is wrong. This last `|` with nothing afterwards creates an empty alternative that will always match

Reply:
Good catch, thanks! 🎉